### PR TITLE
Add post-renderer option to helm-gke-deploy action and clean up documentation

### PR DESCRIPTION
**Pull request by yannick-roeder**

### CODE REVIEW

File: .github/workflows/helm-gke-deploy.yaml [hunk 1]
```diff
@@ -88,12 +97,14 @@ jobs:
           helm-version: ${{ inputs.helm-version }}
 
       - name: Deploy Helm Chart
-        uses: bakdata/ci-templates/actions/helm-gke-deploy@v1.5.0
```
Comment (yannick-roeder) [inline .github/workflows/helm-gke-deploy.yaml:91]: TODO: Update to proper version

File: actions/helm-gke-deploy/action.yaml [hunk 2]
```diff
@@ -42,18 +48,19 @@ runs:
...
-            ${{ inputs.release-name }} ${{ inputs.chart }}
+          echo "Deploying with single value file"          
+          command="${command} --values ${{ inputs.values-yaml }}"
         fi
+
+        if [ -n "${{ inputs.chart-version }}" ]; then command="${command} --version ${{ inputs.chart-version }}"; fi
+        if [ -n "${{ inputs.post-renderer }}" ]; then command="${command} --post-renderer ${{ inputs.post-renderer }}"; fi
+
+        echo "Deploying with command: ${command}"
+        eval "$command"
```
Comment (torbsto) [inline actions/helm-gke-deploy/action.yaml:66]: As you know I'm not a bash expert, but I thought using `eval` should be avoided, see for example [Google's shell guide](https://google.github.io/styleguide/shellguide.html#eval). You could instead collect all parameters in an array and then execute `helm "${parameters[@]}"`.

Comment (yannick-roeder) [reply actions/helm-gke-deploy/action.yaml]: I did it without an array now. What do you think?

**Review by disrupted — APPROVED** [review]

File: actions/helm-gke-deploy/action.yaml (hunk 2 above)
Comment (yannick-roeder) [reply actions/helm-gke-deploy/action.yaml]: I used an array now

**Review by torbsto — APPROVED** [review]

**Pull request merged**